kubernetes/samples/scripts/create-soa-domain/create-soa-domain-pv-pvc/create-pv-pvc.sh

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env bash
```

Comment:
This looks like a copy of the common create-pv-pvc script used by the other samples.  Can this sample similarly use the central create-pv-pvc script?

Reply:
Deleted this directory create-database-pv-pvc.